File: perl/18_multiples_of_a_number.pl
#!/usr/bin/env perl

use strict;
use warnings;

open my $fh, '<', $ARGV[0];

for my $line ( <$fh> ) {
	my @parts = split /,/, $line;
	my $x     = $parts[0];
	my $n     = $parts[1];

	my $v = $n;
	while ( $v < $x ) {
		$v += $n;
	}

	print "$v\n";
}

close $fh;

File: python/18_multiples_of_a_number.py
#!/usr/bin/env python

import sys

with open(sys.argv[1], 'r') as fh:
    for line in fh.readlines():
        line = line.strip()
        if line == '':
            continue

        (x, n) = [int(v) for v in line.split(',')]
        v = n
        while v < x:
            v += n

        print v

